paymentsheet/src/test/resources/paypal-support.csv

```diff
@@ -0,0 +1,25 @@
+lpm, hasCustomer, allowsDelayedPayment, intentSetupFutureUsage, intentHasShipping, intentLpms, supportCustomerSavedCard, formExists, formType, supportsAdding
```

Comment:
Is supportsAdding false here because it is not yet exposed as a supported payment method?

Reply:
yup. I'll have to change this once I enable it as a supported method.